src/Raven.Client/ServerWide/Operations/UpdateDocumentsCompressionConfigurationOperation.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Net.Http;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Documents.Operations;
+using Raven.Client.Http;
+using Raven.Client.Json;
+using Raven.Client.Json.Serialization;
+using Raven.Client.Util;
+using Sparrow.Json;
+
+namespace Raven.Client.ServerWide.Operations
+{
+    class UpdateDocumentsCompressionConfigurationOperation : IMaintenanceOperation<DocumentCompressionConfigurationResult>
+    {
+        private readonly DocumentsCompressionConfiguration _documentsCompressionConfiguration;
+
+        public UpdateDocumentsCompressionConfigurationOperation(DocumentsCompressionConfiguration documentsCompressionConfiguration)
+        {
+            _documentsCompressionConfiguration = documentsCompressionConfiguration ?? throw new ArgumentNullException(nameof(documentsCompressionConfiguration));
+        }
+        public RavenCommand<DocumentCompressionConfigurationResult> GetCommand(DocumentConventions conventions, JsonOperationContext context)
+        {
+            return new UpdateDocumentCompressionConfigurationCommand(_documentsCompressionConfiguration);
+        }
+
+        private class UpdateDocumentCompressionConfigurationCommand : RavenCommand<DocumentCompressionConfigurationResult>, IRaftCommand
+        {
+            private readonly DocumentsCompressionConfiguration _documentsCompressionConfiguration;
+
+            public UpdateDocumentCompressionConfigurationCommand(DocumentsCompressionConfiguration documentsCompressionConfiguration)
```

Comment:
let's simplify the parameter name, rename it to `configuration`

Reply:
Done